src/libp2p_crypto.erl

```diff
@@ -12,9 +12,21 @@
 -export([generate_keys/0, mk_sig_fun/1, load_keys/1, save_keys/2,
          pubkey_to_address/1, address_to_pubkey/1,
          address_to_b58/1, b58_to_address/1,
-         pubkey_to_b58/1, b58_to_pubkey/1, verify/3
+         pubkey_to_b58/1, b58_to_pubkey/1, verify/3,
+         nonl/1
         ]).
 
+-spec nonl(String1) -> String2 when
```

Comment:
Since this is only used in tests, could you move the nonl function to test_util?

Reply:
I meant _use_ the one in test_util